### PR TITLE
Ticket/a22 ab1 12

### PR DESCRIPTION
**Pull request by ghost**
email validation and phone number validation check is added

### CODE REVIEW

**Comment by ghost** [comment]
route to create a new warehouse and inside it validation checks added

File: routes/NewWarehouse/createNewWarehouse.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+router.post("/", function (req, res) {
+const validPhone = req.body.contact.phone.split("").length === 10
+  if (
+    req.body.name &&
+    req.body.address &&
+    req.body.city &&
+    req.body.country &&
+    req.body.contact.name &&
+    req.body.contact.position &&
+   validPhone&&
```
Comment (kevinlinpt) [inline routes/NewWarehouse/createNewWarehouse.js]: nitpick: indentation and spacing between validPhone and &&

File: routes/NewWarehouse/createNewWarehouse.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    );
+    return res.status(201).send(savedWarehouses);
+  } else if(!validator.is_email_valid(req.body.contact.email)) {
+    return res
+      .status(400)
+      .send(`Please provide the valid email addresss`);
+  } else if(!validPhone) {
+    return res.status(400).send("please provide a valid phone number");
+  }else{
```
Comment (kevinlinpt) [inline routes/NewWarehouse/createNewWarehouse.js]: nitpick: spacing between } and else

**Review by kevinlinpt — APPROVED** [review]
Only 2 minor nitpicks, otherwise looks ready to merge

**Pull request merged**